### ISSUE TITLE
MTMamba： Reproduced Training Results Significantly Worse Than Reported in Paper

### ISSUE THREAD

**Issue by aTeppei** [EnVision-Research/MTMamba#4]
**In NYUDv2 Datatset best one：**
{
    "semseg": {
        "mIoU": 0.2695323023250387
    },
    "depth": {
        "rmse": 0.6722854642227621,
        "log_rmse": 0.23594696012281213,
        "abs_rel": 0.18228580031496544,
        "sq_rel": 0.15196204774914554
    },
    "normals": {
        "mean": 23.373225001927377
    },
    "edge": {
        "loss": 0.04809971126394534
    },
    "epoch": 483,
    "best_imp": -0.2966596950610064
}
**and my batch size =8  ，max-iter =50000**
This is my result 
**while in your paper：**
![image](https://github.com/user-attachments/assets/17c1c091-1068-49b5-9f26-1c35a049088b)
Could you tell me more details about the experiment？Thanks for your help！

**Comment by Baijiong-Lin** [comment]
I have retrained and achieved comparable results. Do you successfully use the pre-trained weights in the backbone? I found a minor bug in `pretrained_ckpts/run.sh` and have fixed it now. You can run it to transform the pre-trained weights again if necessary.

**Comment by aTeppei** [comment]
Thanks for your help！
After rectifying the code errors, it is currently operating normally. However, there remains a query. Regarding the task of boundary detection, the loss is employed as an indicator during edge training, but the indicator presented in the paper is odsF. Is there any conversion formula between the two?  
<img width="282" alt="image" src="https://github.com/user-attachments/assets/8fa15ebf-97a1-4f7d-bb07-02ecc473a904" />
![image](https://github.com/user-attachments/assets/b41bf198-8bda-4f62-a7fd-16a0696be203)

**Comment by Baijiong-Lin** [comment]
Following previous work, odsF for boundary detection task is computed by using the third-party tool https://github.com/prismformore/Boundary-Detection-Evaluation-Tools.

**Comment by aTeppei** [comment]
Thanks for your help！
